services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/ManeuverModifier.java

```diff
@@ -0,0 +1,87 @@
+package com.mapbox.api.directions.v5.models;
+
+import androidx.annotation.StringDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Constants for the {@link StepManeuver#modifier()}
+ */
+public interface ManeuverModifier {
+
+    /**
+     * Indicates "uturn" maneuver modifier to be rendered in the [com.mapbox.navigation.ui.instruction.maneuver.ManeuverView]
+     *
+     * @since 5.2.0
+     */
+    String UTURN = "uturn";
```

Comment:
```suggestion
    public static final String UTURN = "uturn";
```

Same thing happens to the rest 👇 

For reference you can take a look at the Java version https://github.com/mapbox/mapbox-navigation-android/blob/e232efd36700e4f5f3345bd76ad99cb28076bb1e/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationConstants.java#L203-L224

Reply:
yeah made it an interface since but think people consider it bad practice 
https://stackoverflow.com/questions/2659593/what-is-the-use-of-interface-constants

can switch